.evergreen/.evg.yml

```diff
@@ -688,3 +705,10 @@ buildvariants:
   run_on: ubuntu1804-test
   tasks:
     - name: "publish-snapshot"
+
+# Storage engine tests on RHEL 6.2 (x86_64)
+- matrix_name: "tests-storage-engines"
+  matrix_spec: { auth: "noauth", ssl: "nossl", jdk: "*", platform: "rhel62", version: ["3.6", "4.0"], topology: ["replicaset", "sharded"], storage-engine: "mmapv1" }
+  display_name: "Storage ${storage-engine} ${jdk} ${platform} ${topology}"
+  tasks:
+    - name: "test"
```

Comment:
Do all the tests pass with mmapv1? If not, we could add a new task here that runs just the single new test.  Similar to gssapi-auth-test

Reply:
Recent changes allow the storage tests to pass with mmapv1.